examples/roll.py

```diff
@@ -0,0 +1,70 @@
+"""An example of a simple CLI for rolling dice.
+
+The example demonstrates the usage of a typed argument
+accepting a list of custom objects (instances of `Die`).
+
+Usage:
+    $ python roll.py 1d6
+    3
+    $ python roll.py 2d6
+    2 5
+    $ python roll.py 1d6 2d3
+    6
+    2 1
+"""
+
+import re
+import random
+from typing import List
+
+from candies.cli import cli, Arg
+
+
+class Die:
+    """Represents a die to roll."""
```

Comment:
I think the 'Represents' word here could be easily removed.

Reply:
Agree, removed.